src/components/slider/slider.tsx

```diff
@@ -165,6 +167,9 @@ export class Slider {
         }
 
         this.createMDCSlider();
```

Comment:
I was checking [MDC's html structure](https://m2.material.io/components/sliders/web#sliders), and saw that they have a simple html template like this:
```html
<div class="mdc-slider">
  <input class="mdc-slider__input" type="range" min="0" max="100" value="50" name="volume" aria-label="Continuous slider demo">
  <div class="mdc-slider__track">
    <div class="mdc-slider__track--inactive"></div>
    <div class="mdc-slider__track--active">
      <div class="mdc-slider__track--active_fill"></div>
    </div>
  </div>
  <div class="mdc-slider__thumb">
    <div class="mdc-slider__thumb-knob"></div>
  </div>
</div>
```

We are wrapping this in two other divs, and then inside a shadow dom.

This might actually be the reasons that their own inbuilt observers (or whatever script they have to calculate the location of the knob) are not doing what they should do. 

Give me some time. I'll try to refactor the component with less nested structure, and see if that has any effect. Then we can continue from there, if things don't work.

Reply:
that didn't help